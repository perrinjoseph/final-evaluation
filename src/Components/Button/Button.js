import React from "react";
import $ from "jquery";
function Button({ content, opposite, scrollTo }) {
  const scrollToInfo = () => {
    window.scroll({
      top: scrollTo,
      left: 0,
      behavior: "smooth",
    });
  };
  return (
    <button
      id="btnInfo"
      onClick={scrollToInfo}
      className={opposite ? "button-opposite" : "button"}
    >
      {content}
    </button>
  );
}

export default Button;
