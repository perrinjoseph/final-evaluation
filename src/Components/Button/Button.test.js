import { render } from "@testing-library/react";
import Button from "./Button";
describe("Button", () => {
  it("should match the snapshot", () => {
    const { asFragment } = render(<Button>Snapshot Button</Button>);
    expect(asFragment()).toMatchSnapshot();
  });
});
