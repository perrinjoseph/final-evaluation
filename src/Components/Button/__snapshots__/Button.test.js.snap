// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match the snapshot 1`] = `
<DocumentFragment>
  <button
    class="button"
  />
</DocumentFragment>
`;
